### PR TITLE
CFD-383: Group management HUB : Add anchors in the group edit form.

### PR DESCRIPTION
**Pull request by YuriGizra**
#575

### CODE REVIEW

**Comment by YuriGizra** [comment]
Output:
selection_124
selection_125
selection_126

ID's on edit group page:
selection_128

**Comment by YuriGizra** [comment]
@nader77 
Ready for review.

File: capacity4more/modules/c4m/features_og/c4m_features_og_manage/c4m_features_og_manage.module [hunk 1]
```diff
@@ -246,3 +246,38 @@ function c4m_features_og_manage_group_status_and_thumbnail() {
...
+ * Custom permission check for group management page.
+ *
+ * Checks if the logged in user has access to "administer group" permission.
+ *
+ * @param $contexts
+ *  The contexts available in the panel.
+ * @return bool
+ *  TRUE when user has access, FALSE when user does not or there's a context missing.
+ */
+function c4m_features_og_manage_page_permissions($contexts) {
```
Comment (nader77) [inline capacity4more/modules/c4m/features_og/c4m_features_og_manage/c4m_features_og_manage.module]: This function was deleted in later fixes, Please delete it from here as well.

File: capacity4more/behat/features/group_manage.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    Then  the response should contain "#edit-details"
+    And   the response should contain "#edit-permissions"
+    And   the response should contain "#edit-related-content"
+    And   the response should contain "#edit-image-banner"
+    And   the response should contain "#edit-image-list"
+
+  @api
+  Scenario: Check anchors in the edit form page
+    Given I am logged in as user "alfrednobel"
+    When  I go to "nobelprize/node/37/edit"
```
Comment (nader77) [inline capacity4more/behat/features/group_manage.feature]: => ``I start editing group "group-name"``

**Comment by nader77** [comment]
@YuriGizra Does not have permission to this repository, This is why this branch is from his forked repo.

**Comment by nader77** [comment]
@YuriGizra Please add a small workflow video as well.

**Comment by YuriGizra** [comment]
a small workflow video:
converted_file_6f9d4af1

**Comment by nader77** [comment]
PR is ready for code review.

**Pull request merged**